check-legacy-endpoint-access/check-legacy-endpoint-access.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: check-legacy-endpoint-access
+  namespace: kube-system
+  labels:
+    app: check-legacy-endpoint-access
+spec:
+  selector:
+    matchLabels:
+      app: check-legacy-endpoint-access
+  template:
+    metadata:
+      labels:
+        app: check-legacy-endpoint-access
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+    spec:
+      hostPID: true
+      containers:
+      - name: check-legacy-endpoint-access
+        image: k8s.gcr.io/startup-script:v2
+        securityContext:
+          privileged: true
```

Comment:
Saw the comments in the original PR. Is there a reason not to use `ubuntu` and get rid of `privileged: true`?

(Not a blocker though.)

Reply:
This may be better than needing to worry about this example pointing to a specific old container version. We don't want to have to update this yaml. I'd do that and go to hostnetwork privs.

Reply:
Hi Yang and Greg,
I use "k8s.gcr.io/startup-script:v2" because it's in the example that I follow and it has curl installed. Now I change to use ubuntu, remove unnecessary privileges  and reformat the output. Please help review it again. Thanks.

Reply:
Hi Yang and Greg,
In previous solution, I install curl at runtime, which is not very good. Now I change to gcr.io/distroless/python3. Could you help review again? Thanks.